### PR TITLE
EREGCSC-1594 -- Fix Vue shorthand errors

### PR DESCRIPTION
**Pull request by PhilR8**
Resolves [EREGCSC-1594](https://jiraent.cms.gov/browse/EREGCSC-1594)

**Description**
The `RelatedRuleList` component was using incorrect named slot shorthand syntax and causing an issue where the Show/Hide buttons were not rendering correctly.

See: https://v2.vuejs.org/v2/guide/components-slots.html#Named-Slots-Shorthand

**This pull request changes**

- Removes colons from v-slot shorthand 
- Adds cypress test to ensure buttons are properly labeled

**Steps to manually verify this change**

1. Visit any Subpart in reader view
2. Expand FR Docs category in right sidebar
3. Make sure the buttons to expand and collapse the FR docs is labeled "+ Show More (X)" and "- Show Less (X)"

### CODE REVIEW

**Comment by PhilR8** [comment]
TODO: perhaps write some tests to cover this UX

File: solution/ui/e2e/cypress/integration/part.spec.js [hunk 1]
```diff
@@ -48,7 +49,11 @@ describe("Part View", () => {
 
         cy.focused().then(($el) => {
             cy.get($el).should("have.id", "433-51");
-            cy.get($el).should("have.css", "background-color", "rgb(238, 250, 254)");
+            cy.get($el).should(
+                "have.css",
+                "background-color",
+                "rgb(238, 250, 254)"
+            );
```
Comment (peggles2) [inline solution/ui/e2e/cypress/integration/part.spec.js:56]: Nice job making it cleaner and more readable.

**Review by peggles2 — APPROVED** [review]
LGTM 👍

File: solution/ui/e2e/cypress/integration/part.spec.js (hunk 1 above)
Comment (PhilR8) [reply solution/ui/e2e/cypress/integration/part.spec.js:56]: Prettier made it prettier!

**Pull request merged**